recipes/lora_finetune_distributed.py

```diff
@@ -287,9 +287,9 @@ def setup(self, cfg: DictConfig) -> None:
         """
         if self._is_rank_zero:
             self._metric_logger = config.instantiate(cfg.metric_logger)
-
             # log config with parameter override
```

Comment:
I think this still needs to log config here. The naming is bad, but one logs it to the metric logger (meaning it's saved into the WandB or Tensorboard run), and the other just prints it to disk